controllers/5-reviews.controllers.js

```diff
@@ -0,0 +1,13 @@
+const { selectReview } = require("../models/5-reviews.models")
+
+exports.manageReviews = (request,response,next) => {
+    const {sort_by, order} = request.query
+   
+    selectReview(sort_by, order).then((result) => { 
+        return response.status(200).send({'reviews': result})
```

Comment:
This is a good response to the client, using `reviews` as the key name is a good choice and represents what's been sent to the client - well done!